Features/links.feature

```diff
@@ -0,0 +1,529 @@
+    En de resource heeft een relatie met één of meer andere resources
+    En de geldigOp parameter wordt meegegeven
+    En de beschikbaarOp parameter wordt niet meegegeven
+    Dan bevat de _links een property voor iedere gerelateerde resource met een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
+    En bevat de link de meegegeven tijdreisparameter geldigOp 
+    En bevat de link de default waarde <DatumTijd van request> voor de tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert één of meerdere resource
+    En de resource heeft een relatie met één of meer andere resources
+    En de geldigOp parameter wordt niet meegegeven
+    En de beschikbaarOp parameter wordt meegegeven
+    Dan bevat de _links een property voor iedere gerelateerde resource met een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
+    En bevat de link de default waarde <Datum van request> voor de tijdreisparameter geldigOp 
+    En bevat de link de meegegeven tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert één of meerdere resource
+    En de resource heeft een relatie met één of meer andere resources
+    En de geldigOp parameter wordt niet meegegeven
+    En de beschikbaarOp parameter wordt niet meegegeven
+    Dan bevat de _links een property voor iedere gerelateerde resource met een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
+    En bevat de link niet de tijdreisparameter geldigOp 
+    En bevat de link niet de tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert één of meerdere resource
+    En een client geeft bepaalde parameters niet op (bv. expand maar niet geldigOp en beschikbaarOp)
+    Dan bevat de _links een property voor iedere gerelateerde resource met een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID, zoals hierboven beschreven (binnnen het scenario)
+    En bevat de link deze parameters niet
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert één of meerdere resource
+    En een client geeft bepaalde parameters niet op (bv. expand maar niet geldigOp en beschikbaarOp)
+    En er wordt een default gebruikt voor deze parameters
+    Dan bevat de _links een property voor iedere gerelateerde resource met een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID, zoals hierboven beschreven (binnnen het scenario)
+    En bevat de link deze parameters niet
+
+  Scenario: links naar gerelateerde resources in een embedded resources
```

Comment:
als hier een vergelijkbaar scenario wordt bedoeld als in regel 23, dan graag dezelfde bewoordingen aanhouden. (collectie en collectie van resources.)

Reply:
Dit zijn twee verschillende scenario's.
Het scenario in regel 23 gaat over de self link van een collectie (hoogste niveau _links in een response).
Dit scenario gaat over de links naar gerelateerde resource in een embedded resource in een resource, waarbij die laatste in een collectie kan zitten (maar dat hoeft niet). Dat is het diepste niveau van links dat we ondersteunen.
Dus er wordt bv. een nummeraanduiding opgevraagd, de openbare ruimte wordt geëxpand en dan wordt hier beschreven hoe de link naar de woonplaats die gerelateerd is aan de openbare ruimte, er uit moet zien.

Reply:
oke